pyicloud/base.py

```diff
@@ -63,7 +64,8 @@ def request(self, method, url, **kwargs):  # pylint: disable=arguments-differ
         # Charge logging to the right service endpoint
         callee = inspect.stack()[2]
         module = inspect.getmodule(callee[0])
-        request_logger = logging.getLogger(module.__name__).getChild("http")
+        name = getattr(module, "__name__") if hasattr(module, "__name__") else __name__
```

Comment:
Can you give me a use case of this ?
Thanks :blush:

Reply:
The use case is safety. When I was using the pyicloud library as an API consumer (maybe in an unsupported manner), w/o this change an error would be raised. In general it's good to be safe.